src/spec.md

```diff
@@ -241,6 +255,9 @@ Webxdc apps run in a restricted environment, but the following practices are per
   e.g. with <https://babeljs.io>
 - `<a href="https://example.org/foo">` and other external links are blocked by definition;
   instead, embed content or use `mailto:` link to offer a way for contact
+- APIs that require user permission may not work, Geolocation, Camera, Microphone etc.
+- `<frame>` or `<iframe>` may not work as expected as used by some implementations for some features
```

Comment:
```suggestion
- None of the features that are enabled through the [Permission Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) work
```

This is about `allow` of `<iframe>`

Reply:
in the rest of the list, we're much more practical an on-point about things not working.

pointing to a multi-page document might be an  an amendment, not a replacement for "frames not working".

and wouldn't the link fit even better to the "APIs that require user permission may not work" point?

so maybe:


``´
- APIs that require [permissions](https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy) may not work, Geolocation, Camera, Microphone etc., see 
- `<frame>` or `<iframe>` may not work as expected as used by some implementations for some features
``´

Reply:
did a commit, let's iterate from there. i think, things are much clearer already, thanks a lot!

Reply:
Maybe, I'm not sure if they're completely the same thing. Also I don't know how it is that

> `<frame>` or `<iframe>` may not work as expected

I didn't say it in #66

Reply:
from #66 

> I believe the fact that [we run in the app in an <iframe>](https://github.com/deltachat/deltachat-desktop/blob/b46b770dbf41dda1406bfa775beae2c6fe2a604a/static/webxdc_wrapper.html#L33-L34) might cause permission-related problems, see e.g. the allow attribute of <iframe>, but I can't name an example.

and i share this statement. we're only saying frames may not work as expected, and are discouraged, not more, not less :) (frames do not make much sense for webxdc anyways as you cannot include foreign content - and for same-source content, there are better approaches)

Reply:
to move forward:

do you think, the two statement: 

- functions requiring permissions may not work
- frames may not work

are wrong?

Reply:
Oh, I see. It's not what I meant, sorry. I mean that the `<iframe>` that we load `index.html` in doesn't have the `allow` attribute, so the entire app, and not just nested `<iframes>` inside of it, probably can't use thing that can be allowed with the `allow` attribute.

Reply:
i see, as being vague, to move forward, we can remove the `<frame> or <iframe> may not work as expected` statement; we can always iterate. this would be pretty much your initial suggestion then.